src/rulesets/Base/array.jl

```diff
@@ -278,3 +278,129 @@ function rrule(::typeof(fill), value::Any, dims::Int...)
     end
     return fill(value, dims), fill_pullback
 end
+
+#####
+##### `findmax`, `maximum`, etc.
+#####
+
+for findm in (:findmin, :findmax)
+    findm_pullback = Symbol(findm, :_pullback)
+
+    @eval function frule((_, xdot), ::typeof($findm), x; dims=:)
+        y, ind = $findm(x; dims=dims)
+        ydot = (xdot[ind], NoTangent())
+        return (y, ind), Tangent{typeof((y, ind)),typeof(ydot)}(ydot)
```

Comment:
Not sure this is correct use of Tangent. Many inscrutable test errors, not sure they are tester or testee errors.